crates/autopilot/src/run.rs

```diff
@@ -543,21 +548,35 @@ pub async fn run(args: Arguments) {
         max_winners_per_auction: args.max_winners_per_auction,
         max_solutions_per_solver: args.max_solutions_per_solver,
     };
+    let drivers_futures = args
+        .drivers
+        .into_iter()
+        .map(|driver| async move {
+            match infra::Driver::new(
+                driver.url,
+                driver.name.clone(),
+                driver.fairness_threshold.map(Into::into),
+                driver.submission_account,
+            )
+            .await
+            {
+                Ok(driver) => Some(Arc::new(driver)),
+                Err(_) => None,
+            }
```

Comment:
Even if the error remains, the match case can be simplified.
```suggestion
            infra::Driver::new(
                driver.url,
                driver.name.clone(),
                driver.fairness_threshold.map(Into::into),
                driver.submission_account,
            )
            .await
            .map(Arc::new)
            .ok()
```